packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -306,6 +307,14 @@ class AppBar extends StatefulWidget implements PreferredSizeWidget {
   /// if that is also null then [ThemeData.primaryIconTheme] is used.
   final IconThemeData iconTheme;
 
+  /// The color, opacity, and size to use for trailing app bar icons. This
+  /// should only be used when the trailing icons should be themed differently
+  /// than the leading icons.
+  ///
+  /// If this property is null then [ThemeData.appBarTheme.actionsIconTheme] is
```

Comment:
null, then

Reply:
done, and made the change to other properties too